questionpy_sdk/watcher.py

```diff
@@ -73,18 +81,15 @@ def _ignore_event(self, event: FileSystemEvent) -> bool:
         Returns:
             `True` if event should be ignored, otherwise `False`.
         """
-        if isinstance(event, FileOpenedEvent | FileClosedEvent):
+        # only consider file modification events
+        if not isinstance(event, FileDeletedEvent | FileModifiedEvent | FileCreatedEvent | FileMovedEvent):
             return True
 
-        # ignore events events in `dist` dir
-        relevant_path = event.dest_path if isinstance(event, FileSystemMovedEvent) else event.src_path
-        if isinstance(relevant_path, bytes):
-            relevant_path = relevant_path.decode()
+        # for move events we want to look at dest_path
+        path_str_or_bytes = event.dest_path if isinstance(event, FileSystemMovedEvent) else event.src_path
+        path_str = path_str_or_bytes.decode() if isinstance(path_str_or_bytes, bytes) else path_str_or_bytes
```

Comment:
Sollte eine nicht ignorierte Datei in eine ignorierte Datei umbenannt werden, ignorieren wir das Event. Dass dann die nicht ignorierte Datei fehlt ist doch aber schon relevant, oder? Wäre für diesen Fall nicht `self._ignore_path(src_path) and self._ignore_path(dest_path)` ein besseres Kriterium?